.github/workflows/release.yaml

```
@@ -0,0 +1,82 @@
+on:
+  push:
```

We could do this to trigger only for pushes to `main` branch
```suggestion
  push:
    branches:
      - main
```